campaign_info.toml

```diff
@@ -6,24 +6,24 @@
 [desktop]
 name = "Desktop"
 icon = "desktop"
-campaign = "C24_WMDE_Desktop_DE_03"
-description = "Based on CTRL of C24_WMDE_Desktop_DE_02"
-campaign_tracking = "03-ba-240808"
+campaign = "C24_WMDE_Desktop_DE_04"
+description = "Test different donation forms"
```

Comment:
It's nice to also have what banner it's based on in the description as when we start jumping tests in the campaign it'll give us an in repo indicator of where a test came from.